extension/ezoe/ezxmltext/handlers/input/ezoeinputparser.php

```diff
@@ -1250,9 +1250,9 @@ function publishHandlerLink( $element, &$params )
                 if ( !eZContentObject::exists( $objectID ))
                 {
                     $this->Messages[] = ezpI18n::tr( 'design/standard/ezoe/handler',
-                                                'Object %1 does not exist.',
+                                                'Invalid link: "%1" target object does not exist.',
```

Comment:
Nitpick: inconsistency with other messages, missing **:** after "%1".

Reply:
Consider improvement like this:
`Invalid link: "%1". Target object does not exist."`